internals/secrethub/generate.go

```diff
@@ -59,13 +60,31 @@ func (cmd *GenerateSecretCommand) Register(r Registerer) {
 }
 
 // before configures the command using the flag values.
-func (cmd *GenerateSecretCommand) before() {
-	cmd.generator = randchar.NewGenerator(cmd.useSymbols)
+func (cmd *GenerateSecretCommand) before() error {
+	useSymbols := false
+
+	useSymbolsEnv := os.Getenv("SECRETHUB_GENERATE_RAND_SYMBOLS")
+	if useSymbolsEnv != "" {
+		b, err := strconv.ParseBool(useSymbolsEnv)
+		if err != nil {
+			return err
+		}
+		useSymbols = useSymbols || b
```

Comment:
What's happening here? It's already set to false, so can we do without the `||` check here and just say `useSymbols = b`?